google-cloud-clients/google-cloud-spanner/src/main/java/com/google/cloud/spanner/DatabaseClient.java

```diff
@@ -85,34 +85,43 @@
 
   /**
    * Returns a context in which a single read can be performed using {@link TimestampBound#strong()}
-   * concurrency.
+   * concurrency. This method will return a {@link ReadContext} that will not return the read
+   * timestamp that was used by Cloud Spanner. If you want to be able to access the read timestamp,
+   * you should use the method {@link #singleUseReadOnlyTransaction()}.
    *
    * <p>Example of single use.
    *
-   * <pre>{@code
```

Comment:
Do we need to remove `@code`?

Reply:
It's no removed, but changed by the formatter into '&#64;code'.

Reply:
That's odd - I tried running it myself and didn't see the same output (stayed the same). Can you try resubmitting as `@code` and see if the linter passes?

Reply:
The wrong formatting was first introduced by my IDE (Eclipse), and the mvn plugin did not automatically change it back to the original value. It's now changed back to `@code`.